tests/base.py

```diff
@@ -0,0 +1,333 @@
+import time
+from tap_tester.scenario import (SCENARIOS)
```

Comment:
@harshpatel4crest Where are we leveraging this import statement?

Reply:
Removed that statement